docs/database/database-spans.md

```diff
@@ -458,7 +458,7 @@ name or target).
     command.CommandText = "some_stored_procedure";
     ```
 
-    the corresponding `db.query.summary` is `EXECUTE some_stored_procedure`, `db.query.text` is not populated.
+    the corresponding `db.query.summary` is `CALL some_stored_procedure`, `db.query.text` is not populated.
```

Comment:
@alanwest raised a good question about this being under a SQLServer example where maybe it should be EXECUTE

Reply:
Here's some links for the purpose of leaving some breadcrumbs

* T-SQL's (SQL Server's SQL dialect) documentation for its [EXECUTE statement](https://learn.microsoft.com/sql/t-sql/language-elements/execute-transact-sql)
* Helpful [StackOverflow post](https://stackoverflow.com/questions/7846418/what-is-the-difference-between-call-and-exec-in-t-sql) on this topic.

Reply:
Something like

```suggestion
    the corresponding `db.query.summary` is `EXECUTE some_stored_procedure`, `db.query.text` is not populated. The `EXECUTE` statement is SQL Server's means to invoke a stored procedure. If the database system uses a different keyword for executing a stored procedure then that keyword SHOULD be used (e.g., `CALL some_stored_procedure`). 
```